config/example-extras.cfg

```diff
@@ -583,6 +583,38 @@
 #a0_pin
 #   The pins connected to an uc1701 type lcd. These parameters must be
 #   provided when using an uc1701 display.
+#menu_root:
+#   Entry point for menu, root menuitem name. This parameter must be
+#   provided when using menu.
+#menu_timeout:
+#   Timeout for menu. Being inactive this amount of seconds will trigger
+#   menu exit or return to root menu when having autorun enabled [optional]
+#menu_autorun:
+#   Enable this to run menu immediately at startup. It's mostly used to replace
+#   stock info screens with custom ones. It accepts boolean values,
+#   default is False. [optional]
```

Comment:
As a high-level comment, I don't think we should document the autorun capability until such time as it is ready for regular use.  At that time, we should replace the current display code with the new, more flexible, display code.  I fear regular users will just be confused by having two overlapping mechanisms that implement the same thing.